### PR TITLE
rm list of contributors

### PR DESCRIPTION
**Pull request by MarcoFalke**
The list was severely outdated and could potentially create the impression that there is a closed set of contributors to the the open source software client.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
See discussion in  #470

File: _data/contributors.yml [hunk 1]
```diff
@@ -23,116 +23,11 @@ maintainers:
...
-    role:
-  -
-    name: paveljanik
-    github: paveljanik
-    irc: paveljanik
-    twitter:
-    avatar: https://avatars1.githubusercontent.com/u/6848764?v=3&s=460
-    bio:
-    role:
+    role: Wallet maintainer
```
Comment (jnewbery) [inline _data/contributors.yml]: 😕

**Comment by MarcoFalke** [comment]
@jnewbery I am not objected to a replacement of the team page that makes more sense. However, I think first removing the current team page is orthogonal to that and an improvement on its own.

(Also amended the section to what you suggested in the comment you just linked to)

**Comment by jnewbery** [comment]
> orthogonal

I don't think this word means what you think it means :)

**Comment by jnewbery** [comment]
utACK c8618b1359005d78b584c95b73d3856a84c52736

**Comment by luke-jr** [comment]
This seems to only get rid of some of the team, and not others, giving an erroneous impression that merge permissions is more important than it really is. IMO either the page should be removed entirely, or otherwise updated to a current list. NACK removing some active people and leaving others...

**Comment by MarcoFalke** [comment]
Done

**Comment by luke-jr** [comment]
Okay, concept ACK then I guess. (Not really setup to render/test it myself.)

**Comment by jnewbery** [comment]
> giving an erroneous impression that merge permissions is more important than it really is

Not really - the difference is that a maintainer has a well-defined role in the github organization. They have permissions to merge code. I don't think identifying them infers that they're any more or less 'important'.

Concept ACK removing all, or just leaving maintainers.

**Comment by jl2012** [comment]
concept ACK remove all

**Comment by harding** [comment]
Concept ACK for remove all, but I suggest we leave a redirect to another page on the site like the [contribute page](https://bitcoincore.org/en/contribute/) so the URL isn't broken.  For instructions on creating the redirect, see https://github.com/bitcoin-core/bitcoincore.org/pull/518#issuecomment-369680701

**Comment by achow101** [comment]
Concept ACK

**Comment by laanwj** [comment]
I kind of like the idea of giving frequent contributors a more visible place then at the bottom of release notes, if they want. But I also agree that if it doesn't get updated it's better to remove it.
Also we already have the team pages on the bitcoin / bitcoin-core organizations, and keeping these things updated in parallel is a pain.

**Comment by MarcoFalke** [comment]
@harding Can you elaborate a bit on that. I think I only removed files in the `_data` and `_include` directories and only reformulated the actual `_post`. Not sure which page needs a redirect...

**Comment by laanwj** [comment]
I think he means that the URL should lead to a redirect in case external sites might be pointing as that's clearer than showing a not-found page.

**Comment by MarcoFalke** [comment]
What does "the URL" refer to?

On Mar 3, 2018 13:46, "Wladimir J. van der Laan" <notifications@github.com>
wrote:

I think he means that the URL should lead to a redirect in case external
sites might be pointing as that's clearer than showing a not-found page.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/bitcoin-core/bitcoincore.org/pull/519#issuecomment-370170400>,
or mute the thread

**Comment by harding** [comment]
@MarcoFalke sorry, I thought this PR removed the team page but I just confirmed that it only removes the list of contributors from that page.

Tested ACK 41bd570ae68fb8831a584eb816df371d2236e99f

Preview of the updated page:

**Pull request merged**